src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -759,32 +756,15 @@ export abstract class AbstractSettingRenderer extends Disposable implements ITre
 
 		this.ignoredSettings = getIgnoredSettings(getDefaultIgnoredSettings(), this._configService);
 		this._register(this._configService.onDidChangeConfiguration(e => {
-			if (e.affectedKeys.includes('settingsSync.ignoredSettings')) {
-				this.ignoredSettings = getIgnoredSettings(getDefaultIgnoredSettings(), this._configService);
-				this._onDidChangeIgnoredSettings.fire();
-			}
+			this.ignoredSettings = getIgnoredSettings(getDefaultIgnoredSettings(), this._configService);
```

Comment:
Note: we fire the event more often here by taking out the if case, because otherwise, OSS doesn't seem to update `this.ignoredSettings` when the extension settings finally get loaded.